resources.rc

```diff
@@ -1,4 +1,36 @@
 #include "resources.h"
 IDR_ICON              ICON                      "icon.ico"
 IDR_PEN               CURSOR                    "pen.cur"
-IDR_ERASER            CURSOR                    "eraser.cur"
\ No newline at end of file
+IDR_ERASER            CURSOR                    "eraser.cur"
+
+#define VER_MAJOR 1
+#define VER_MINOR 0
+#define VER_REVISION 7
+
+
+
+#define INNER_TO_STR(x) #x
+#define TO_STR(x) INNER_TO_STR(x)
+#define VER_STRING TO_STR(VER_MAJOR) "." TO_STR(VER_MINOR) "." TO_STR(VER_REVISION) ".0"
+
+#define VS_VERSION_INFO 0x1L
+
+VS_VERSION_INFO VERSIONINFO
+FILEVERSION     VER_MAJOR,VER_MINOR,VER_REVISION
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904E4"
+        BEGIN
+            VALUE "FileDescription",  "Symphony Screen Snippet Tool"
+            VALUE "ProductName",      "Symphony Screen Snippet Tool"
+            VALUE "ProductVersion",   VER_STRING
+            VALUE "LegalCopyright",    "Copyright 2020 Symphony OSS"
```

Comment:
any chance we can make the year dynamic here?

Reply:
Not easily - we would need a tool to generate a .h file containing #defines for the dynamic information. It would be nice to have this for version as well (taken from package.json). But would take some work...